proto/node/akash/market/v1/event.proto

```
@@ -115,3 +115,28 @@ message EventLeaseClosed {
+    (gogoproto.moretags)   = "yaml:\"id\""
+  ];
+
+  // reason is the provider's stated reason for reclamation.
+  LeaseClosedReason reason = 2 [
+    (gogoproto.jsontag)  = "reason",
+    (gogoproto.moretags) = "yaml:\"reason\""
+  ];
+
+  // deadline is the unix timestamp when the reclamation window expires.
+  int64 deadline = 3 [
+    (gogoproto.jsontag)  = "deadline",
```

Buf (1.68.4)

[error] cannot find `gogoproto.jsontag` in this scope

(COMPILE)